frontend/src/app/store/reducers/auth-page.reducers.ts

```diff
@@ -12,6 +12,7 @@ export const initialState: AuthState = {
 
 export const authPageReducer = createReducer(
     initialState,
-    on(register, (state, { user }): AuthState => ({ ...state, user })),
-    on(verifyEmail, (state, { token }): AuthState => ({ ...state, token }))
+    on(addUser, (state, { user }): AuthState => ({ ...state, user })),
+    on(addToken, (state, { token }): AuthState => ({ ...state, token })),
+    on(login, (state, { authState }): AuthState => (authState))
```

Comment:
Set exact fields and expect exact class for action. Tommorow you will add to authState some field and won't notice that login response does not return it at all and will just rewrite store, lose new field value.

Reply:
Fixed